### PR TITLE
Bump master to 3.11

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by beav** [comment]
@ehelms are these kind of PRs ok to merge without redmine issues?

File: lib/katello/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Katello
-  VERSION = "3.10.0-master".freeze
+  VERSION = "3.11.0-master".freeze
```
Comment (beav) [inline lib/katello/version.rb]: This needs to just be numbers for `Gem::Dependency` to work I think. For example:

```
irb(main):007:0> Gem::Dependency.new('', ' >= 3.7.0').match?('', '3.10.0')
=> true
irb(main):008:0> Gem::Dependency.new('', ' >= 3.7.0').match?('', '3.10.0.pre.master')
=> false
```

**Comment by ehelms** [comment]
I'd prefer not to remove master. This is the master branch and as you can
see Rubygems has support for including a pre release info into the version.
We added this to more clearly match our packaging and to indicate it's
bleeding edge nature

**Comment by beav** [comment]
@ehelms i think i see:

```
irb(main):002:0> Gem::Dependency.new('', ' >= 3.7.0.pre.master').match?('', '3.10.0')
=> true
irb(main):003:0> Gem::Dependency.new('', ' >= 3.7.0.pre.master').match?('', '3.10.0.pre.master')
=> true
irb(main):004:0> Gem::Dependency.new('', ' >= 3.7.0').match?('', '3.10.0.pre.master')
=> false
```

It looks like the matcher will not say that 3.10.pre.master is higher than 3.7.0 unless you say `prerelease = true`. I'll work on it more as part of the other issue, removing nack

**Comment by ehelms** [comment]
Thanks!

**Pull request merged**